Resources/ConfigPresets/WizardsDen/wizardsDenGateway.toml

```diff
@@ -0,0 +1,16 @@
+# Configuration preset layer for gateway Wizard's Den servers.
+# Intended to be loaded after wizardsDen.toml
+
+# hostname is a placeholder
+# needs region tag
+
+[game]
+hostname = "[EN] Wizard's Den Farm Grass Hopper"
+desc = "Official English Space Station 14 servers for new players. Vanilla, roleplay ruleset."
+panic_bunker.enabled = false
+panic_bunker.disable_with_admins = false
+panic_bunker.enable_without_admins = false
+panic_bunker.custom_reason = ""
```

Comment:
There is a PR up for allowing to do inverse panic bunkers (maximum playtime)
#28763